[DCOS-54188] Release Elastic and Kibana 6.8.1. (#2501)
mesosphere/dcos-docs-site: pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/release-notes/index.md

-   Upgraded SDK version from `0.55.4` to `0.56.2`.

    Please take a look at the following SDK change logs for more information:

    -   [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)
    -   [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
    -   [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
    -   [0.56.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.2)

    Changes: ([#34](https://github.com/mesosphere/dcos-elastic-service/pull/34))
-   Replaced Oracle JRE (jre-8u192) with newer OpenJDK (openjdk-jre-8u212b03). ([#3060](https://github.com/mesosphere/dcos-commons/pull/3060))

<a id="orgce810fb"></a>

## New Features

-   Added support for Elasticsearch plugin installation via proxy. New settings are:

    -   `elasticsearch.plugin_http_proxy_host`
    -   `elasticsearch.plugin_http_proxy_port`
    -   `elasticsearch.plugin_https_proxy_host`
    -   `elasticsearch.plugin_https_proxy_port`

    Changes: ([#4](https://github.com/mesosphere/dcos-elastic-service/pull/4))

> -   Added custom domain support for Elastic. New setting is `service.security.custom_domain`. This custom domain is used in place of the default `autoip.dcos.thisdcos.directory`. This can be used to expose the service securely outside of the cluster, but requires setting up external DNS. Check [this documentation page](https://mesosphere.github.io/dcos-commons/developer-guide/#externalizing-transport-encryption-and-security) for more information. ([#9](https://github.com/mesosphere/dcos-elastic-service/pull/9))

Reviewer: Same comment regarding adding a section for custom domains. Similar to [this](https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/security/#forwarding-dns-and-custom-domain)

-   Made it possible to scale down the amount of coordinator nodes. ([#15](https://github.com/mesosphere/dcos-elastic-service/pull/15))

> -   Added custom YAML configuration support to Kibana. New setting is: `kibana.custom_kibana_yml` ([#14](https://github.com/mesosphere/dcos-elastic-service/pull/14))

Reviewer: perhaps add a small section for this in the docs as well? Since you have it for elasticsearch [here](https://github.com/mesosphere/dcos-docs-site/pull/2501/files#diff-cc2f50e1be0ae6d2e06ad57a6b173940R111)

-   Added plugin installation support for Kibana. New setting is: `kibana.plugins` ([#16](https://github.com/mesosphere/dcos-elastic-service/pull/16))

> -   Added volume profile support. New settings are:

Reviewer: Might be worth adding a separate section in the `Configuration` or `Advanced` page to show users how to set `volume_profiles` to configure DSS. That way, users can find this info in future docs and not have to refer to these release notes that might not exist). I've made the same recommendation in C*, HDFS, and Kafka docs (like [here](https://github.com/mesosphere/dcos-docs-site/pull/2486/files#diff-a24d0015c230a5d609b3c0e1d11461a1R70) and [here](https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/configuration/#using-volume-profiles))

-   Added placement constraints capability to Kibana. New setting is `kibana.placement` (array of placement constraint arrays). Check the [Marathon placement constraints documentation](https://mesosphere.github.io/marathon/docs/constraints.html) for more details. ([#27](https://github.com/mesosphere/dcos-elastic-service/pull/27))
-   Added virtual network (CNI) support to Kibana. New settings are:

    -   `virtual_network_enabled` (boolean)
    -   `virtual_network_name` (string, default `dcos`)
    -   `virtual_network_plugin_labels` (array of objects with `key` and `value` keys)

    Changes: ([#28](https://github.com/mesosphere/dcos-elastic-service/pull/28))
-   Added scheduler-plan based health checks. ([#3069](https://github.com/mesosphere/dcos-commons/pull/3069))

<a id="orgae15112"></a>

## Bug Fixes

-   Fixed Kibana service URL (`<cluster-url>/service/kibana`). This makes Kibana's service URL work out of the box even when X-Pack Security is enabled. This was a limitation in the previous version, where we required Kibana to be exposed with EdgeLB. Which is still possible, but not required anymore. ([#3036](https://github.com/mesosphere/dcos-commons/pull/3036))

> -   Fixed issue where setting `http.cors.allow-origin` to `*` would cause the deploy to fail to due the Elasticsearch configuration file be compiled with incorrectly escaped content. ([#23](https://github.com/mesosphere/dcos-elastic-service/pull/23))

Reviewer: ```suggestion
-   Fixed issue where setting `http.cors.allow-origin` to `*` would cause the deployment to fail due to the Elasticsearch configuration file being compiled with incorrectly escaped content. ([#23](https://github.com/mesosphere/dcos-elastic-service/pull/23))
```
